example.py

```diff
@@ -37,8 +37,10 @@ def main():
     print()
     time.sleep(1)
 
+    model = constants.Model.CLAUDE_OPUS
```

Comment:
Change the model to `sonnet` since most people wont have access to opus. I'd also change the default model everywhere to `sonnet`.